### ISSUE TITLE
I want a rule that will catch `@pytest.mark.django_db()()`

### ISSUE THREAD

**Issue by sobolevn** [m-burst/flake8-pytest-style#262]
# Rule request

## Description

This code was allowed: https://github.com/tough-dev-school/python-testing-homework/commit/352aa9b27d17d25f9aa6d67ddb4bec03503ae326

So, I propose to forbid it:

```python
@pytest.mark.whatever()()
def test_health_check() -> None: ...
```

Double call is clearly not correct.
